libiocage/cli/__init__.py

```diff
@@ -95,11 +142,16 @@ def get_command(self, ctx, name):
                 pass
             return mod.cli
         except (ImportError, AttributeError):
+            raise
             return
 
 
+@click.option("--log-level", "-d", default=None)
 @click.command(cls=IOCageCLI)
-@click.version_option(version="0.9.10 07/30/2017", prog_name="iocage",
+@click.version_option(version="0.2.11 08/29/2017", prog_name="ioc",
```

Comment:
I started with 0.0.1 and everytime it will introduce new features I'd love to bump the minor version. For breaking changes (after 1.0.0) the major version number will increase. We should discuss and document the versioning system!